src/mesh/Mesh.cpp

```diff
@@ -263,9 +263,9 @@ void Mesh::clear()
   _tetrahedra.clear();
   _index.clear();
 
+  clearDataStamples();
   for (mesh::PtrData &data : _data) {
     data->values().resize(0);
-    data->timeStepsStorage().clear();
   }
 }
```

Comment:
Is the change of order here problematic?

`data->values().resize(0)` is something we would like to remove in the long term but currently I would be careful with changing the order of the statements if it is not necessary. There are sometime strange side-effects when the storage and the values interact with each other.

Reply:
There are no side effects here and this should hopefully go away soon.